src/bin/rational.rs

```diff
@@ -479,6 +601,33 @@ impl SynthLanguage for Math {
 
             let lvec = Self::eval_pattern(lhs, &env, n);
             let rvec = Self::eval_pattern(rhs, &env, n);
+
+            // if we are writing cvecs to a file
+            if synth.params.write_ces {
```

Comment:
ohh, i see, so writing to the file is a separate feature from using the cvecs from a file. Ok this makes sense i think. You can ignore my previous comment on why we have both, but do consider renaming them to be a bit more indicative of what they are used for. Maybe `persist_cvec` is a better name for this flag